configdefinitions/src/vespa/dispatch.def

```diff
@@ -48,6 +48,9 @@ numJrtTransportThreads int default=8
 # Number of JRT connections per backend node
 numJrtConnectionsPerNode int default=8
 
+# Number of seconds to spend warming up code to prevent JIT cold start issues.
+warmuptime double default=0.1
```

Comment:
Any particular reason for the double level of defaults? Just set 5 here?

Reply:
Intention to avoid long unit test runtime.